networks.Rmd

```diff
@@ -0,0 +1,259 @@
+
+```{r}
+graph %>% 
+  activate(nodes) %>% 
+  mutate(centrality = centrality_pagerank()) %>% 
+  arrange(desc(centrality))
+```
+
+### Want more?
+
+This is just a brief glimpse into tidygraph, for the sake of understanding ggraph. If you are interested in learning more, the tidygraph website gives an overview of all the functionalities in the package: <https://tidygraph.data-imaginist.com>
+
+## Visualizing networks
+
+ggraph builds on top of tidygraph and ggplot2 to allow a complete and familiar grammar of graphics for network data. Still, it is a little different from most ggplot2 extension packages since it works with another data type that is fundamentally different from tabular data. More so, most network visualizations don't concern themselves with mapping variables to `x` and `y` aesthetics since they are concerned with shoving the network topology more than relations between two variables. In order to show network topology the concept of layouts are employed. Layouts are algorithms that use the network structure to calculate (often arbitrary) `x` and `y` values for each node that can then be used for visualization purpose.
+
+### Setting up the visualization
+
+Whereas a normal ggplot2 plot is initialized with a `ggplot()` call, a ggraph plot is initialized with a `ggraph()` call. The first argument is the data, which can be a tbl\_graph or any object convertible to one. The second argument is a layout function and any further argument will be passed on to that function. The default layout will choose an appropriate layout based on the type of graph you provide, but while it is often a descent starting point you should always take control and explore the different layouts available --- networks are nefarious for their ability to show non-existing or exaggerated relations in some layouts.
+
+#### Specifying a layout
+
+The layout argument can either take a string or a function. If a string is provided, the name will be matched to one of the build in layouts (of which there are many). If a function is provided it is assumed that the function takes a tbl\_graph and returns a data frame with at least an x and y column and with the same number of rows as there are nodes in the input graph. Below we can see examples of using the default layout, specifying a specific layout, and providing a layout function directly:
```

Comment:
I think you can leave the use of a function for the docs, and instead show another existing layout.